src/diffusers/models/controlnet.py

```diff
@@ -830,7 +830,6 @@ def forward(
                 sample = self.mid_block(sample, emb)
 
         # 5. Control net blocks
-
```

Comment:
make style or accident?

Reply:
This was intentional, can revert if not needed. I thought there was an extra line between the comment and code by mistake